CONTRIBUTING.md

```diff
@@ -378,6 +380,15 @@ et my_machine -t="8000:8000"
 
 Then navigate to `localhost:8000` in your web browser.
 
+Alternatively, you can run `scp` on your local machine to copy the files from
+your remote machine:
+```bash
+mkdir -p build cpp/build
+scp -r me@my_machine:/path/to/pytorch/docs/build/html build
+scp -r me@my_machine:/path/to/pytorch/docs/cpp/build/html cpp/build
```

Comment:
maybe suggest to package those folders into tar and scp down? would that make it faster?

Reply:
oh good point! I'll try that

Reply:
I ended up [using `rsync`](https://unix.stackexchange.com/a/238207/311046) instead of `scp`, and it is indeed much faster :)